src/main/scala/extracells/gui/GuiFluidStorage.java

```diff
@@ -198,18 +202,19 @@ protected void mouseClicked(int mouseX, int mouseY, int mouseBtn) {
 	}
 
 	public void updateFluids() {
+		if (this.searchbar != null && !Objects.equals(this.searchbar.getText(), "")) {
```

Comment:
minor:
```suggestion
		if (this.searchbar != null && !"".equals(this.searchbar.getText())) {
```

Reply:
I think this logic should also be in forceFluidUpdate, not in the callers

Reply:
```suggestion
		if (this.searchbar != null && !this.searchbar.getText().isEmpty()) {
```
Is a preferred way.
If `searchbar.getText()` can possibly be null then look at `net.minecraft.util.StringUtils#isNullOrEmpty(String)V`